### DISCUSSION TITLE
Memory Leaks

### DISCUSSION THREAD

**Discussion by Raidansz** [pointfreeco/swift-composable-architecture#3362] [General]
### Description

I was testing the demo app provided by you at
https://github.com/pointfreeco/episode-code-samples/tree/main/0245-tca-tour-pt3/Standups
and found out that there are memory leaks via using the profiler. You can run the profiler and check for leaks and you will get the same result.
<img width="387" alt="Screenshot 2024-09-07 at 7 26 11 PM" src="https://github.com/user-attachments/assets/a5498afa-e405-457e-83d7-515603b092e2">

### Checklist

- [x] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-composable-architecture/issues) or [discussion](https://github.com/pointfreeco/swift-composable-architecture/discussions).

**Comment by mbrandonw** [comment]
Hi @Raidansz, I only see mentions of SwiftUI symbols in your screenshot, and so it is not clear to me that there is an issue with the library. If you can dig up more information and find definitive proof that there is a leak in the library, then we will be happy to look into it.

I am going to convert this to a discussion for now. Please feel free to continue the conversation over there.
